python/cudf/cudf/core/dataframe.py

```diff
@@ -488,6 +484,51 @@ def __getitem__(self, arg):
             msg = "__getitem__ on type {!r} is not supported"
             raise TypeError(msg.format(type(arg)))
 
+    def where(self, cond, other):
+        """
+        Replace values with other where the condition is False.
+
+        Parameters
+        ----------
+        cond : boolean
```

Comment:
This should be an array-like of booleans or a Frame of booleans and we should note that callables are not yet supported